packages/sandbox/src/file_watching_sandbox.ts

```diff
@@ -87,9 +87,19 @@ export class FileWatchingSandbox extends EventEmitter implements Sandbox {
    * @inheritdoc
    */
   start = async (options: SandboxOptions) => {
-    this.filesChangesTracker = await createFilesChangesTracker(
-      options.dir ?? './amplify'
-    );
+    const watchDir = options.dir ?? './amplify';
+    options.dir && path.isAbsolute(options.dir)
+      ? options.dir
+      : path.join(process.cwd(), options.dir ?? './amplify');
+    if (!fs.existsSync(watchDir)) {
+      this.printer.log(
+        `${watchDir} does not exist. please try running this command from your project root`,
+        LogLevel.ERROR
+      );
+      return;
+    }
```

Comment:
Should we throw `AmplifyUserError` here instead?
@Amplifiyer what do you think ?

Reply:
I think this is a good suggestion. If you try to start sandbox pointing to a path that doesn't exist, it should just fail.

Reply:
throwing is a good idea that should result in the exit code 1. Just returning here would end up exiting as success.

Reply:
@sobolk @edwardfoyle @Amplifiyer 
Thank you for your a good suggestion.
I change that throw `AmplifyUserError` instead of printing error log and return.